python_modules/dagster/dagster/_core/definitions/graph_definition.py

```diff
@@ -808,6 +808,13 @@ def tags(self) -> Mapping[str, str]:
         """The tags associated with the graph."""
         return super().tags
 
+    @property
+    def pools(self) -> set[str]:
```

Comment:
nit: prefer `from collections.abc import Set`

Reply:
is that everyone's preference?  the incidence of `set` is like 4x the count of `Set`:

```
(dagster) ➜  dagster git:(prha/asset_op_concurrency_key) s '\bset\[' | wc
     423    2458   58005
(dagster) ➜  dagster git:(prha/asset_op_concurrency_key) s '\bSet\[' | wc
      99     773   15437
```

Reply:
for typing yeah -- I imagine almost all of those lower case `set`s are at runtime

until recently this was `from typing import AbstractSet`, `from collections.abc import Set` is the same thing but is now preferred by our linter -- this marks the return value as immutable, which has some value, but just in general I haven't seen `set[x]` used as a return type anywhere else in the code so we should be consistent